edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -726,6 +726,11 @@ impl ModuleRuntime for DockerModuleRuntime {
                             version: edgelet_core::version_with_source_version(),
                             provisioning,
                             cpus: system_info.NCPU().unwrap_or_default(),
+                            virtualized: match edgelet_core::is_virtualized_env() {
```

Comment:
Generally it would be better to use an enum here, rather than Some<bool>. That way more options can be added in the future and more information can be sent if needed, since enums can contain other values. This id fine for now, but may need to change in the future.

Reply:
Okay.